### PR TITLE
Remove overhang logo and user when viewing media

### PR DESCRIPTION
**Pull request by cewert**
### Why remove the logo?
Branding is good but too much branding is bad. At the very minimum, the user will still see the logo on the home screen of the app as well as the server and user select screens. This just removes the logo from screens that display media (the home screen shows media too but is more of a dashboard) to allow the focus to be on the media itself. The user knows what app they are using at this point.

### Why remove the user?
As mentioned above, the home screen is basically the dashboard for the app and this is the one screen where the currently logged in user is shown. If the user ever isn't sure who is the logged in user, then can navigate to the home screen. This also allows more focus to be on the media itself while viewing it.

Before:
unstable-logo

After:
remove-logo

## Changes
- Crop logo images to remove wasted space around the image
- Add overhang logo to "left group"
- Hide overhang logo and user when navigating away from the home screen

### CODE REVIEW

**Comment by cewert** [comment]
It looks like the image quality went down a bit when I cropped these in paint 😢 We should recreate the images from the SVG sources.

**Comment by sevenrats** [comment]
this is also more in line with web, which is nice.
do you mind attaching a screenshot of the home screen with the modified logo?

File: source/utils/misc.bs [hunk 1]
```diff
@@ -507,3 +507,42 @@ function shuffleArray(array as object) as object
...
     return array
 end function
+
+' Create and return a Jellyfin logo poster node
+function createLogoPoster()
+    logoPoster = createObject("roSGNode", "Poster")
+    logoPoster.id = "overlayLogo"
+    logoPoster.uri = "pkg:/images/logo.png"
+    logoPoster.translation = "[70, 53]"
+    logoPoster.width = "270"
```
Comment (1hitsong) [inline source/utils/misc.bs]: These are the old height and width values, not the new cropped values.

File: components/JFOverhang.xml [hunk 2]
```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <component name="JFOverhang" extends="Group">
   <children>
-    <Poster id="overlayLogo" uri="pkg:/images/logo.png" translation="[70, 53]" width="270" height="72" />
-    <LayoutGroup id="overlayLeftGroup" layoutDirection="horiz" translation="[375, 53]" itemSpacings="30">
-      <Rectangle id="overlayLeftSeperator" color="#666666" width="2" height="64" />
+    <LayoutGroup id="overlayLeftGroup" layoutDirection="horiz" translation="[54, 54]" itemSpacings="60">
+      <Poster id="overlayLogo" uri="pkg:/images/logo.png" height="66" />
```
Comment (1hitsong) [inline components/JFOverhang.xml]: The logo looks stretched horizontally. I had to add the width of 191 to make the image look normal.

File: components/config/SetServerScreen.bs [hunk 3]
```diff
@@ -156,3 +156,15 @@ end function
 sub clearErrorMessage()
     m.top.errorMessage = ""
 end sub
+
+' JFScreen hook.
+' Used to show logo, update the focus, the state of the data, and tells the server about the device profile.
```
Comment (1hitsong) [inline components/config/SetServerScreen.bs]: Event handler function descriptions should describe the event being handled, not the functionality it contains. This prevents function descriptions from becoming stale or overloaded.

File: components/home/Home.bs [hunk 4]
```diff
@@ -30,9 +30,17 @@ sub loadLibraries()
     m.fadeInFocusBitmap.control = "start"
 end sub
 
-' JFScreen hook that gets ran as needed.
-' Used to update the focus, the state of the data, and tells the server about the device profile
+' JFScreen hook.
+' Used to show logo, update the focus, the state of the data, and tells the server about the device profile.
```
Comment (1hitsong) [inline components/home/Home.bs]: Event handler function descriptions should describe the event being handled, not the functionality it contains. This prevents function descriptions from becoming stale or overloaded.

File: components/home/Home.bs [hunk 5]
```diff
@@ -53,6 +61,18 @@ sub OnScreenShown()
     end if
 end sub
 
+' JFScreen hook.
+' Remove the Jellyfin logo and left seperator from the overhang
+sub OnScreenHidden()
```
Comment (1hitsong) [inline components/home/Home.bs:64]: Event handler function descriptions should describe the event being handled, not the functionality it contains. This prevents function descriptions from becoming stale or overloaded.

File: components/login/UserSelect.bs [hunk 6]
```diff
@@ -24,6 +24,17 @@ sub redraw()
     m.top.findNode("UserRow").translation = [leftBorder, topBorder]
 end sub
 
+' JFScreen hook.
+' Used to show logo, update the focus, the state of the data, and tells the server about the device profile.
+sub OnScreenShown()
```
Comment (1hitsong) [inline components/login/UserSelect.bs:28]: Event handler function descriptions should describe the event being handled, not the functionality it contains. This prevents function descriptions from becoming stale or overloaded.

**Review by 1hitsong — CHANGES_REQUESTED** [review]

File: components/home/HomeRows.bs [hunk 7]
```diff
@@ -94,7 +94,12 @@ sub processUserSections()
     ' Add home sections in order based on user settings
     loadedSections = 0
     for i = 0 to 6
-        sectionName = LCase(m.global.session.user.settings["homesection" + i.toStr()])
+        sectionName = m.global.session.user.settings["homesection" + i.toStr()]
+        if isValid(sectionName)
+            sectionName = LCase(sectionName)
+        else
+            exit for
+        end if
```
Comment (1hitsong) [inline components/home/HomeRows.bs]: We can simplify and remove the else clause.

```
if not isValid(sectionname) then exit for

sectionName = LCase(sectionName)
```

**Review by 1hitsong — CHANGES_REQUESTED** [review]
Functionality looks good now. 1 small code style change.

**Review by 1hitsong — APPROVED** [review]

**Pull request merged**